docs/concepts/deployments.md

```diff
@@ -306,7 +360,7 @@ $ prefect deployment inspect 'Cat Facts/catfact'
 
 ## Running deployments
 
-If you specify a schedule for a deployment, the deployment will execute its flow automatically on that schedule as long as a Prefect Orion API server and agent is running.
+If you specify a schedule for a deployment, the deployment will execute its flow automatically on that schedule as long as a Prefect Orion API server and agent is running. Prefect Cloud can create scheduled flow runs automatically as long as an agent is configured to pick up flow runs for the deployment.
```

Comment:
The new sentence is not quite right -- the scheduler will run no matter what and create new runs, but they won't actually run unless an agent is running that can pick them up.